paf-mvp-benchmark/K6/samples/signed-prefs.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I don't understand why there are both a json file and a constant in `data.js`, can you please justify?